### DISCUSSION TITLE
Utilizing Multi-Regression in V2: Seeking Guidance

### DISCUSSION THREAD

**Discussion by Gavision97** [chemprop/chemprop#885] [Q&A]
My objective is to perform a multi-regression task where there are multiple targets to predict. I couldn't find any documentation on this. Can someone help?

**Comment by KnathanM** [comment]
Hi @Gavision97, sorry you had trouble finding that in the documentation. I assume you are using Chemprop in a notebook and not from the command line. There are two things you need to do for multi-regression. 

First when you make datapoints, the targets are given as 1D numpy array. `datapoint = MoleculeDatapoint(mol=mol, y=np.array([1, 2, 3])).

Second, when you make the feed forward network, you need to tell it how many tasks it will predict. `ffn = RegressionFFN(n_tasks=3).

Let me know if you have further questions
